app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -1548,4 +1550,15 @@ class StatePlayerRecyclerViewAssembler private constructor(
       }
     }
   }
+
+  /** Saves the expanded state to a Bundle. */
+  fun saveExpandedState(outState: Bundle) {
+    outState.putBoolean(HAS_PREVIOUS_RESPONSES_EXPANDED_KEY, hasPreviousResponsesExpanded)
+  }
+
+  /** Restores the expanded state from a Bundle. */
+  fun restoreExpandedState(savedInstanceState: Bundle?) {
+    hasPreviousResponsesExpanded =
+      savedInstanceState?.getBoolean(HAS_PREVIOUS_RESPONSES_EXPANDED_KEY, false) ?: false
+  }
```

Comment:
We have moved to using protos only for arguments.(We refactored all usages of getInt, Boolean, String etc). PTAL at https://github.com/oppia/oppia-android/blob/6e349bc9f21a685861214c6164adbc1af78f30fa/app/src/main/java/org/oppia/android/app/topic/lessons/TopicLessonsFragment.kt#L78-L89 for reference. The protos are created in `arguments.proto`.